spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
It's definitely undesirable to have hard coded id refs.  That's why I first tried to implement using the value_for_label() logic.  What I ran into was the factory seems to be getting triggered before the seeds.rb was executing.  

I'm thinking now that there should be a way to better handle the seeds.rb invoke generically from the spec/rails_helper.rb.  Thoughts?